dao/authentication_dao.go

```diff
@@ -398,6 +435,51 @@ func authFromVault(secret *api.Secret) *m.Authentication {
 		auth.SourceID = id
 	}
 
+	// Request the JWT token if the authentication is a "marketplace" authentication
+	if auth.Name == "marketplace" {
```

Comment:
After another look - what do think about moving branch `auth.Name == "marketplace"` to a function ?

Reply:
Initially I thought about extracting it to a function, but since `authFromVault` is already the third nested function that gets called, I thought that having to go yet into another function would make it harder to understand.

However now I'm unsure about it :S

Let me try to develop this and send another commit to see how it looks

Reply:
yea that's right, 
cool!